tests/test_tacacs_server_group_style2.rb

```diff
@@ -0,0 +1,151 @@
+#
+# Minitest for TacacsServerGroup class
+#
+# Copyright (c) 2014-2016 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require File.expand_path('../ciscotest', __FILE__)
+require File.expand_path('../../lib/cisco_node_utils/tacacs_server_group', \
+                         __FILE__)
+
+# TestTacacsServerGroup - Minitest for TacacsServerGroup node utility.
+class TestTacacsServerGroup < CiscoTestCase
```

Comment:
Same comment as the other PR:

Can you rename this class to something unique (e.g. TestTacacsServerGroupStyle2)? Having the class named the same as the class in test_tacacs_server_group.rb is a bit confusing (difficult to tell which test was run by looking at the output).